objects/cash_register.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+
+class CashRegister
+  def total
+    @value = 0.0
+  end
+
+  def showing
+    @value
+  end
+
+  def purchase
```

Comment:
The `purchase` method is supposed to take an argument `amount` and add the amount to the total.